Development practices/Background and scheduled jobs.md

```diff
@@ -0,0 +1,98 @@
+Our goal is to have applications that process incoming web requests in a timely fashion.
+Some requests don't just manipulate data from your database, but can:
```

Comment:
According to https://painintheenglish.com/case/4723 it's either `e.g.` or `eg`, so I'll go with the latter :)

Reply:
I won't split hairs regarding this, but in this case and in others I'd rather check more reliable sources like the Oxford Dictionary (https://www.oxfordlearnersdictionaries.com/definition/english/e-g) because they're the authority in this field and things then aren't left for interpretation.